lib/schema/manifest.go

```diff
@@ -869,10 +869,10 @@ type Volume struct {
 	Mode string `json:"mode,omitempty"`
 	// Recursive means that all mount points inside this mount should also be mounted
 	Recursive bool `json:"recursive,omitempty"`
-	// Label specifies the SELinux label.
+	// SELinuxLabel specifies the SELinux label.
 	// If left unspecified, the default.ContainerFileLabel will be used to label the directory.
-	// If a special value NoLabel is specified - no labeling is performed
-	Label string `json:"label,omitempty"`
+	// If a special value SELinuxLabelNone is specified - no labeling is performed
+	SELinuxLabel string `json:"seLinuxLabel,omitempty"`
```

Comment:
Per your & @r0mant's advice to me in https://github.com/gravitational/robotest/pull/193#discussion_r398268442, should this be a pointer to a string to distinguish between the unset and "none" case?

I assume you chose to use a string flag for a reason, and I'd like to understand why/how this is different than the uid/gid settings case.

Reply:
In case of uid/gid which were represented as integers and the fact that `0` is a valid value for both it made it ambiguous to tell between unset and `0`.
In case of strings, the empty value (or unset) cannot be taken for a valid value. Also "none" being a special value vs empty is a design decision meaning no labeling vs default label.